src/components/widgets/sales-opportunities-funnel/sales-opportunities-funnel.directive.coffee

```diff
@@ -34,13 +33,17 @@ module.controller('WidgetSalesOpportunitiesFunnelCtrl', ($scope, $q, ChartFormat
   w.initContext = ->
     if $scope.isDataFound = angular.isDefined(w.content) && !_.isEmpty(w.content.opps_per_sales_stage) && hasOneOpportunity(w.content.opps_per_sales_stage)
 
-      $scope.statusOptions = _.compact _.map w.metadata.sales_stage_selection, (status) ->
```

Comment:
**warning**, name of the param here is "sales_stage_selection" and not "status_selection": changing this would simply break the widget as the backend is not designed to receive a "status_selection" metadata parameter...